packages/volto-slate/src/editor/config.jsx

```diff
@@ -207,7 +207,7 @@ export const hotkeys = {
   'mod+b': { format: 'strong', type: 'inline' },
   'mod+i': { format: 'em', type: 'inline' },
   'mod+u': { format: 'u', type: 'inline' },
-  'mod+s': { format: 'del', type: 'inline' },
+  'mod+d': { format: 'del', type: 'inline' },
```

Comment:
I would also not recommend using just ctrl+d as it's a bookmark shortcut and interferes with the browser. Gutenberg uses opt+ctrl+d in osx and shfit+alt+d in windows. Maybe you could take inspiration from that or else.

Reply:
Thanks both of you for your insights :). I forgot about bookmark shortcut.
As JeffersonBledsoe and others suggested, `mod+shift+x` would be best then.